### DISCUSSION TITLE
i Can´t Start

### DISCUSSION THREAD

**Discussion by mofde2003** [pcolby/dokit#23] [Q&A] [answered]
Hey, can you tell me how to start it? For me the console only opens briefly and closes or am I doing something wrong here?

Kind regards Marcel

**Comment by pcolby** [comment, marked as answer]
Hi @mofde2003, what OS are you using?

I'm guessing Windows, in which case you probably want to use a Windows Command Prompt (aka `cmd.exe`) to start with, like:

1. [Open a Command Prompt](https://www.pcworld.com/article/395081/open-command-prompt-in-windows.html) (any those 11 ways should be fine)
2. Run: `cd <dir-where-you-downloaded-dokit>`
3. Run: `dokit.exe --help`

Cheers.
